### ISSUE TITLE
Sidebar notifications for new emails not working

### ISSUE THREAD

**Issue by yanntrividic** [tutao/tutanota#3977]
Hello,

I picked the desktop label but this issue is cross-platform. I'm using Tutanota on Android, Ubuntu and Windows and all three show the same behavior. A few weeks ago, I exported a copy of a several emails folders that I then deleted. It resulted in the client not showing if a new mail was received: notifications are not shown, and there is no number appearing next to the folder's name in the sidebar (the only exception seems to be is with the `Spam` folder, the `Inbox` folder is also having this problem). 

In order to check if I have received a new email, I have to check for each folder one at a time (those folders are managed with inbox rules). The new emails are marked as unread and are highlighted correctly, but nothing appears in the sidebar.

I don't know how to reproduce the issue as I don't know how to fix it. I tried to log out all my devices, and to update all Tutanota clients. It's still there.

 - OS: Ubuntu 20.04, Windows 10, Android 10 QKQ1.190828.002
 - Desktop Environment: Gnome 3.36.8
 - Tutanota version: 3.91.10

It could be important to mention that I also changed the folders' names to make them appear in the order I want. To bypass the alphabetical order, I used `_` and `!`. For example, `_b` appears before `!a` in the sidebar. I tried renaming those folders to their former name but the issue is still there.

**Comment by charlag** [comment]
Hi
Do I understand correctly that you don't see correct counters next to the folders inside the app?

**Comment by charlag** [comment]
Does Websocket connect work for you? Do you see other changes reflected in the app e.g. when you move an email do you see it move?

**Comment by yanntrividic** [comment]
Yes, I see it move. The only malfunction really is those missing counters... Although it seems to be getting better: I've been able to notice some counters appearing in the past few days (but not all of them).

**Comment by charlag** [comment]
Well that's not a lot of info to work off from. Counters are received altogether: either all of them should be there or none.

**Comment by yanntrividic** [comment]
Interesting... Well I'll try to observe this behavior as much as I can and update this thread if I notice anything new, or manage to solve it.

**Comment by sofiageo** [comment]
Hi, I have the same issue currently for one of my folders. This folder has **two rules** and they are both on the same domain, subdomain. Other folders work fine with either multiple rules on different domains, or with single rules. I've only noticed it on the web interface, because I'm mainly using this one. I might be wrong but I think it used to be working fine until July 23.

**Comment by yanntrividic** [comment]
Hi, in reaction to @sofiageo, here is an update of my issue: the same problem kept happening for a few weeks and then just disappeared. Everything is working fine know. The disappearance of this problem wasn't preceded by the installation of an update. I can edit my original post if needed.

**Comment by sofiageo** [comment]
I also checked the response of the GET request after I change the problematic email to unread, and the only difference I can see is that "differentEnvelopeSender" is null. I'm not sure if that field is used for the sidebar counter.

**Comment by sofiageo** [comment]
FYI this is still happening for random emails.

![Screenshot 2022-12-24 152546](https://user-images.githubusercontent.com/6191600/209438126-fa451915-e5f0-4589-9240-656ce6c22e0d.png)
